### DISCUSSION TITLE
Convert APBS OpenDX Grid to XYZ Format

### DISCUSSION THREAD

**Discussion by sbembenek18** [Electrostatics/apbs#284] [General] [answered]
I'm trying to convert the APBS OpenDX grid to XYZ format along with the electrostatic potential value. This would be useful for other calculations I want to do.

Has anyone done this before?

**Comment by sobolevnrm** [comment, marked as answer]
Hello --

The code that should help you do this is available at https://github.com/Electrostatics/apbs/blob/c2e5af22c9044b2bb8f9c1114710add58ec5b007/src/mg/vgrid.c#L179.

Thanks,

Nathan

**Reply by sbembenek18** [reply]
Thanks!
